app/uk/gov/hmrc/gform/formtemplate/FormComponentMakerService.scala

```diff
@@ -47,9 +47,9 @@ object FormComponentMakerService {
     toUpperCase: UpperCaseBoolean) = (maybeFormatExpr, maybeValueExpr, displayWidth) match {
     // format: off
     case (Some(TextFormat(f)), HasTextExpression(expr), None)                => Text(f, expr, DisplayWidth.DEFAULT, toUpperCase).asRight
-    case (None,                HasTextExpression(expr), None)                => Text(ShortText, expr, DisplayWidth.DEFAULT, toUpperCase).asRight
+    case (None,                HasTextExpression(expr), None)                => Text(ShortText(), expr, DisplayWidth.DEFAULT, toUpperCase).asRight
```

Comment:
Let's not rely on default values like in `ShortText()` at all. Let's introduce something like `ShortText.default` instead.

```
object ShortText {
  val default = ShortText(0, 1000)
}
```

Ask @Davidrymer97 how the other day we've spent whole morning hunting for a bug which turned out to be caused by default value in a case class.